### PR TITLE
Translation status view

### LINKED ISSUE #5300: need a view of which content is translated into which language
Goal is to be able to see all campaign and static content and the status of what language that content exists in.

bonus points if we can group it by content type. also if doing two different content types is too complex, let's just stick with campaign content.

| NID | TRANSLATE | US EN | GL EN | BR PT | MX SP |
| --- | --- | --- | --- | --- | --- |
| 32 | [13 Gallon Challenge](https://thor.dosomething.org/node/32/translate) | Published | Not translated | Not translated | Not published |
| 5459 | [GTFO](https://thor.dosomething.org/node/5459/translate) | Published | Published | Published | Published |
| 5626 | [Editor Created Test](https://thor.dosomething.org/node/5626/translate) | Not published | Published | Not Published | Not Published |

maybe this could be a new tab on /admin/content as "Translation Status"

only administrators and Global Admins should have access to this

### PR DESCRIPTION
**Pull request by itsjoekent**
#### What's this PR do?

Adds a new view to the /admin/content/search/translation which lists all campaigns and there associated translations 
#### How should this be manually tested?

navigate to the above URL ^ and check the data in the table is correct
#### Any background context you want to provide?

@angaither and I opted to make a quick n' dirty solution to this that doesn't look to great rathter than coding & theming this custom. @mikefantini approved the v1 version
#### What are the relevant tickets?

Fixes #5300 

@blisteringherb can you verify the exports?
@DFurnes can you verify the css?

EDIT: also ffs of course theres a merge conflict

### CODE REVIEW

**Comment by DFurnes** [comment]
screen shot 2015-10-05 at 1 47 27 pm

**Comment by itsjoekent** [comment]
yeah I noticed.. but is the css :ok: ?

File: lib/modules/dosomething/dosomething_user/dosomething_user.module [hunk 1]
```diff
@@ -1452,4 +1452,10 @@ function dosomething_user_views_pre_render(&$view) {
   if ($view->name == 'user_search' && count($view->result) == 1) {
     drupal_goto('user/' . $view->result[0]->uid);
   }
+
+  if ($view->name == 'content_search') {
+    // Remove grouping labels
+    $table_style = 'table caption { display: none }';
+    drupal_add_css($table_style, $option['type'] = inline);
```
Comment (DFurnes) [inline lib/modules/dosomething/dosomething_user/dosomething_user.module]: Should this be `drupal_add_css($table_style, ['type' => 'inline']);`? The syntax you used looks a little funky.

Comment (itsjoekent) [inline lib/modules/dosomething/dosomething_user/dosomething_user.module]: mmm, yeah probably. i just copied this from somewhere else

**Comment by DFurnes** [comment]
Looks good to me! :art:

**Comment by blisteringherb** [comment]
:+1:

**Pull request merged**